doc/source/ray-core/actors/scheduling.rst

```diff
@@ -25,10 +25,22 @@ See :ref:`Placement Group <ray-placement-group-doc-ref>` for more details.
 
 Scheduling Strategy
 -------------------
-Actors support ``scheduling_strategy`` option to specify the strategy used to decide the best node among available nodes.
-Currently the only supported strategy for actors is "DEFAULT".
-"DEFAULT" is the default strategy used by Ray. With the current implementation, Ray will try to pack actors on nodes
+Actors support a ``scheduling_strategy`` option to specify the strategy used to decide the best node among available nodes.
+Currently the supported strategies for actors are ``"DEFAULT"`` and ``ray.util.scheduling_strategies.NodeAffinitySchedulingStrategy(node_id, soft: bool)``.
+
+``"DEFAULT"`` is the default strategy used by Ray. With the current implementation, Ray will try to pack actors on nodes
 until the resource utilization is beyond a certain threshold and spread actors afterwards.
+
+NodeAffinitySchedulingStrategy is a low-level strategy that allows an actor to be scheduled onto a particular node specified by its hexadecimal node id.
```

Comment:
let's omit the `hexadecimal` here -- from the user's perspective, there should be only one node id and they shouldn't need to worry if it is hexadecimal or binary etc.